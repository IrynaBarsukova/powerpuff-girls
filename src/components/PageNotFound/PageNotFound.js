import React from 'react';

import styles from './PageNotFound.scss';

function PageNotFound() {
    return (
        <div className={styles.pageNotFound}>
            <h2 className={styles.pageNotFound__title}>Page not found</h2>
        </div>
    );
}

export default PageNotFound;
