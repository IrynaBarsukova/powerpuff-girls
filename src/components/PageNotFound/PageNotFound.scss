.pageNotFound {
  height: 100vh;
  display: flex;
  justify-content: center;
  align-items: center;
}

.pageNotFound__title {
  font-weight: normal;
}
