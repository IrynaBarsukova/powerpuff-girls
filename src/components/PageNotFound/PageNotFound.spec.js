import React from 'react';
import { render } from '@testing-library/react';

import PageNotFound from './PageNotFound';

describe('<PageNotFound />', () => {
    it('should render PageNotFound', () => {
        const { container } = render(<PageNotFound />);

        expect(container).toMatchSnapshot();
    });
});
