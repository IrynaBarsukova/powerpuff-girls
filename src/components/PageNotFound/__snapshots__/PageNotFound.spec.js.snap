// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageNotFound /> should render PageNotFound 1`] = `
<div>
  <div
    class="pageNotFound"
  >
    <h2
      class="pageNotFound__title"
    >
      Page not found
    </h2>
  </div>
</div>
`;
